### PR TITLE
Cmdlets renaming and Aliases

### PR DESCRIPTION
**Pull request by v-akarke**

### CODE REVIEW

**Review by SteveMutungi254 — DISMISSED** [review]
LGTM.

File: module/Entra/AdditionalFunctions/Get-EntraDirectoryObjectOnPremisesProvisioningError.ps1 [hunk 1]
```diff
@@ -35,4 +35,6 @@ function Get-EntraHasObjectsWithDirSyncProvisioningError {
         }
         
     }
-}
\ No newline at end of file
+}
+
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Get-EntraDirectoryObjectOnPremisesProvisioningError.ps1:39]: These extra lines are unnecessary

File: module/Entra/AdditionalFunctions/Update-EntraUserFromFederated.ps1 [hunk 2]
```diff
@@ -78,4 +78,6 @@ function Convert-EntraFederatedUser {
         }
         $response
         }
-}
\ No newline at end of file
+}
+
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Update-EntraUserFromFederated.ps1:82]: These extra lines are unnecessary

File: module/Entra/AdditionalFunctions/Get-EntraDirectoryObjectOnPremisesProvisioningError.ps1 (hunk 1 above)
Comment (v-akarke) [reply module/Entra/AdditionalFunctions/Get-EntraDirectoryObjectOnPremisesProvisioningError.ps1]: Removed!

File: module/Entra/AdditionalFunctions/Update-EntraUserFromFederated.ps1 (hunk 2 above)
Comment (v-akarke) [reply module/Entra/AdditionalFunctions/Update-EntraUserFromFederated.ps1]: Removed extra lines!

**Review by KenitoInc — APPROVED** [review]

**Review by KenitoInc — APPROVED** [review]

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**